autorest/codegen/templates/enum.py.jinja2

```diff
@@ -6,5 +6,8 @@ class {{ enum.name }}(with_metaclass(_CaseInsensitiveEnumMeta, {{ enum.enum_type
     {% endif %}
 
     {% for value in enum.values %}
-    {{ value.name }} = {{ enum.enum_type.get_declaration(value.value) }}{{ "  #: " + value.description if value.description else "" }}
+    {% if value.description %}
```

Comment:
Do we have test to cover this logic?
It's worth to add tests so that we can have more confidence to refactor/refine/fix our code.

Reply:
I mentioend this in the info box, but I currently don't have a test for it, I do link what it. looks like generated though. SInce you're asking, I might as well add a test, will update this PR with it